apps/backend/internal/presentation/api/handler/payment.go

```diff
@@ -122,3 +105,45 @@ func (ph *paymentHandler) Create(c echo.Context) error {
 
 	return c.JSON(201, res)
 }
+
+func (ph *paymentHandler) Get(c echo.Context, id string) error {
+	input := usecase.GetPaymentInput{
+		ID: id,
+	}
+	event, err := ph.pu.Get(input)
+	if err != nil {
+		message := fmt.Sprintf("internal err: %s", err.Error())
+		res := &api.ErrorResponse{
+			Message: message,
+		}
+		return c.JSON(http.StatusInternalServerError, res)
+	}
+
+	var debtors []api.PaymentUserWithAmount
+	for _, d := range event.Debtors() {
+		debtor := api.PaymentUserWithAmount{
+			Id:     d.ID().String(),
+			Name:   d.Name(),
+			Email:  d.Email(),
+			Avatar: d.Avatar(),
+		}
+		debtors = append(debtors, debtor)
+	}
```

Comment:
コレってAmountも必要じゃないすかね？

```suggestion
	var debtors []api.PaymentUserWithAmount
	for _, d := range event.Debtors() {
		debtor := api.PaymentUserWithAmount{
			Id:     d.ID().String(),
			Name:   d.Name(),
			Email:  d.Email(),
			Avatar: d.Avatar(),
      Amount: uint64(d.Amount().Value()), // 追加
		}
		debtors = append(debtors, debtor)
	}
```